src/test/java/org/broadinstitute/hellbender/tools/copynumber/FilterIntervalsIntegrationTest.java

```diff
@@ -27,6 +27,7 @@
 import org.testng.annotations.Test;
 
 import java.io.File;
+import java.lang.reflect.AnnotatedElement;
```

Comment:
Stray import?

Reply:
Hmm not sure how that got in there. Thanks!